doc/060_forget.rst

```diff
@@ -23,13 +23,6 @@ data that was referenced by the snapshot from the repository. This can
 be automated with the ``--prune`` option of the ``forget`` command,
 which runs ``prune`` automatically if snapshots have been removed.
 
-.. Warning::
-
-   Pruning snapshots can be a very time-consuming process, taking nearly
-   as long as backups themselves. During a prune operation, the index is
-   locked and backups cannot be completed. Performance improvements are 
-   planned for this feature.
```

Comment:
:smiley:

Reply:
Here's a suggestion for a new text that might be something of a middle-ground (I agree we shouldn't remove the warning entirely):

~~~
Pruning snapshots can be a time-consuming process, depending on the
amount of snapshots and data to process. During a prune operation, the
repository is locked and backups cannot be completed. Please plan your
pruning so that there's time to complete it and it doesn't interfere with
regular backup runs.
~~~

Reply:
When deleting this passage, #2842 was still included in this PR :smile: 
I took the suggestions of @rawtaz